src/exampleTypes/TagsQuery/utils.js

```diff
@@ -2,18 +2,21 @@ import _ from 'lodash/fp'
 import { bgJoin } from '../../styles/generic'
 import { tagToGroupJoin } from '../TagsJoinPicker'
 
+export const TAG_TERM = 'keyword'
```

Comment:
We generally don't do the all caps thing even for constants - we also don't use `const` because it isn't a true immutable thing. While it could work for strings, we've generally standardized on not using it so we shouldn't do it here either